cmd/crictl/main.go

```diff
@@ -270,12 +271,34 @@ func main() {
 			Usage: "Address to which the gRPC tracing collector will send spans to.",
 			Value: "127.0.0.1:4317",
 		},
+		&cli.StringFlag{
+			Name:  "profile-cpu",
+			Usage: "Write a pprof CPU profile to the provided path.",
+		},
+		&cli.StringFlag{
+			Name:  "profile-mem",
+			Usage: "Write a pprof memory profile to the provided path.",
+		},
 	}
 
 	app.Before = func(context *cli.Context) (err error) {
 		var config *common.ServerConfiguration
 		var exePath string
 
+		cpuProfilePath := context.String("profile-cpu")
+		if cpuProfilePath != "" {
+			logrus.Infof("Creating CPU profile in: %s", cpuProfilePath)
+
+			file, err := os.Create(cpuProfilePath)
+			if err != nil {
+				return fmt.Errorf("could not create CPU profile: %w", err)
```

Comment:
please include path of the file in the error message.

Reply:
Maybe even absolute path we calculated. Since we allow path traversal, it may not be obvious what file we tried to write to.

Reply:
Yes, using the absolute path now.